### PR TITLE
MULE-19987: Change severity level in validation of expressions

### PR DESCRIPTION
**Pull request by marianorean**

### CODE REVIEW

File: src/main/java/org/mule/runtime/api/config/MuleRuntimeFeature.java [hunk 1]
```diff
@@ -181,7 +182,7 @@
   /**
    * When enabled,
    *
-   * @since 4.4.0-202201
+   * @since 4.4.0-202202
```
Comment (elrodro83) [inline src/main/java/org/mule/runtime/api/config/MuleRuntimeFeature.java:185]: why this change?

File: src/main/java/org/mule/runtime/api/config/MuleRuntimeFeature.java [hunk 2]
```diff
@@ -196,7 +197,16 @@
...
-      "MULE-19919", "4.5.0", DEFAULT_ERROR_HANDLER_NOT_ROLLBACK_IF_NOT_CORRESPONDING_PROPERTY);
+      "MULE-19919", "4.5.0", DEFAULT_ERROR_HANDLER_NOT_ROLLBACK_IF_NOT_CORRESPONDING_PROPERTY),
+
+  /**
+   * When enabled, expression validations will be enforced for targetValue, not allowing a literal value.
+   *
+   * @since 4.4.0-202202
+   */
+  ENFORCE_REQUIRED_EXPRESSION_VALIDATION(
+      "When enabled, expression validations will be enforced for targetValue, not allowing a literal value.",
```
Comment (elrodro83) [inline src/main/java/org/mule/runtime/api/config/MuleRuntimeFeature.java:208]: the implementation affects the validation in general, not just for the targetValue parameter.

**Review by elrodro83 — DISMISSED** [review]

**Review by pabloperalta — APPROVED** [review]

**Review by eze210 — APPROVED** [review]
